src/main/java/com/spring/book/management/dto/UserLoginResponseDto.java

```diff
@@ -0,0 +1,13 @@
+package com.spring.book.management.dto;
+
+public class UserLoginResponseDto {
+    private final String jwtToken;
```

Comment:
Rename it to avoid any mismatch if you later change getter name.
```suggestion
    private final String token;
```